src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -3357,6 +3357,37 @@
     <Field Name="Keyword" Type="SyntaxToken" Override="true">
       <ContextualKind Name="RecordKeyword"/>
     </Field>
+    <Field Name="RecordModifier" Type="SyntaxToken" Optional="true">
+      <Kind Name="ClassKeyword"/>
+    </Field>
+    <Field Name="Identifier" Type="SyntaxToken" Override="true">
```

Comment:
It looks like in cases like this we use names specific to the keyword. For example:
```
    <Field Name="UsingKeyword" Type="SyntaxToken">
      <Kind Name="UsingKeyword"/>
    </Field>
```